build/vsts-helpers.ps1

```diff
@@ -13,36 +12,27 @@
 		$Colour
 	)
 	
-	# Store the colour to be able to revert back to it.
-	$previousColour = [System.Console]::ForegroundColor
-	
-	# Set the appropriate foreground text colour.
-  	Invoke-Expression -Command "[System.Console]::ForegroundColor = [System.ConsoleColor]::$Colour"
+	# Initialize the variable to output.
+	$HelpMessage = ''
 	
 	# Add some extra spacing to either side of the message.
 	$length = $Message.Length + 2
 	
-	# Write the top border.
-	Write-Host "+" -NoNewline
-	for ($i = 0; $i -lt $length; $i++)
-	{
-		Write-Host "-" -NoNewline
+	function WriteBorder($remark){
+		$HelpMessage += "+"
+		for ($i = 0; $i -lt $length; $i++)
+		{
+			$HelpMessage += "-"
+		}
+		$HelpMessage += "+$([Environment]::NewLine)" 
 	}
-	Write-Host "+"
+
+	. WriteBorder 'Top'
 	
-	# Write the text.
-	Write-Host "| " -NoNewline
-	Write-Host $Message -NoNewline
-	Write-Host " |"
+	$HelpMessage += "| $Message |$([Environment]::NewLine)" 
 	
-	# Write the bottom border.
-	Write-Host "+" -NoNewline
-	for ($i = 0; $i -lt $length; $i++)
-	{
-		Write-Host "-" -NoNewline
-	}
-	Write-Host "+"
+	. WriteBorder 'bottom'
 	
-	# Set the colour back to "normal".
-	[System.Console]::ForegroundColor = $previousColour
+	# Write the text.
+	Write-Host $HelpMessage -ForegroundColor $Colour
```

Comment:
Now that this is the new way of setting the output colour, the `[ValidateSet()]` attribute should have the `Bold` value changed to `White`, since `Bold` will not be accepted as a `-ForegroundColor`.

Reply:
Yes, I just realized that Bold corresponds to White.

Reply:
![image](https://user-images.githubusercontent.com/38101170/105567536-82127c00-5d6d-11eb-8f43-9d06765b3bb5.png)